jodd-http/src/main/java/jodd/http/net/SocketHttpConnectionProvider.java

```diff
@@ -50,6 +50,7 @@
 
 	protected ProxyInfo proxy = ProxyInfo.directProxy();
 	protected String secureEnabledProtocols = System.getProperty("https.protocols");
+	protected String sslProtocol = "SSL";
```

Comment:
I'd recommend to bump this up to TLSv1.1. While that might break connections to old SSL instances it will provider a better default security and - if really necessary - it can always be downgraded by the respective setter.